README.md

```diff
@@ -17,14 +18,15 @@ public interface AppServerForTesting {
 }
 ```
 
-You will notice that it will use the implementation you set for "healthcheck" into KaryonBootstrap
+You will notice that it will use the implementation you set for "healthcheck" into KaryonBootstrap.
+See AppServerForTesting for further details and a functional implementation.
 
 ## Gradle
 
 Add dependency as follows:
 
 ```
-    compile 'com.scmspain:karyon-health-check-endpoint-module:0.1.0-SNAPSHOT'
+    compile 'com.scmspain.karyon:karyon-healthcheck:0.1.1'
```

Comment:
I think it would be a good idea to add maven central badges to avoid people hardcoding the version seen on documentation.

https://github.com/jirutka/maven-badges